feature_engine/selection/base_selector.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+
+def get_feature_importances(estimator):
+
+    """Retrieve feature importances from a fitted estimator"""
+
+    importances = getattr(estimator, "feature_importances_", None)
+
+    coef_ = getattr(estimator, "coef_", None)
+
+    if importances is None and coef_ is not None:
```

Comment:
Question: could it not be simplified to `if coef_` ?

I think `if coef_is not None`, is captured by `if coef_`, and I don't think there is a scenario where` importances is not None` and `Coef_` is also `not None`, so not sure we need both? I might be wrong.